scene.go

```diff
@@ -64,6 +67,18 @@ func TransitionFade(rate float32, frames int) func(*image.RGBA, int) bool {
 	}
 }
 
+func TransitionZoom(xPerc, yPerc float64, frames int, zoomRate float64) func(*image.RGBA, int) bool {
+	return func(buf *image.RGBA, frame int) bool {
+		if frame > frames {
+			return false
+		}
+		z := render.Zoom(xPerc, yPerc, 1+zoomRate*float64(frame)/2)
```

Comment:
The divide by 2 is concerning. I understand the starting from one (w/1=w). The effect in practice looks right so perhaps I am missing something here as to why you chose to divide by this constant.